src/components/layout/nav/nav-partner.tsx

```diff
@@ -12,16 +11,13 @@ type NavPartnersProps = {
 
 const NavPartners = ({ hide_login_signup }: NavPartnersProps) => {
     return (
-        <>
-            <PartnerWrapper>
-                <MainNav />
-                <PartnerNavigationBarWrapper>
-                    <NavPartnerDesktop hide_login_signup={hide_login_signup} />
-                    <NavPartnerMobile hide_login_signup={hide_login_signup} />
-                </PartnerNavigationBarWrapper>
-            </PartnerWrapper>
-            <CFDWarning />
```

Comment:
Not sure why we have been rendering `CFDWarning` separately again in `NavPartners` and `NavSecurity`, We have just removed them and rendering it only once in the `LayoutOverlay` instead.